HHS/Head-Start-TTADP
# [TTAHUB-404] Add target populations to Grantee record page

## Description of change

Add the target populations list to the Grantee record page

## How to test

View the TTA Profile tab of a grantee. Verify that the widget appears and that the data is correct.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-404


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
